config/locales/en.yml

```diff
@@ -76,3 +76,14 @@ en:
     cookies_text: To be of better service to you this website makes use of cookies
     ok: Accept cookies
     learn_more: More info
+
+  submissions:
+    publishing:
+      publish: |-
+        This submission is not yet visible to other people. Publishing will make it
+        immediately visible to everybody but you will have one week to revoke it
+        in case of a mistake. After one week publication becomes permanent.
+      revoke: |
+        Revocation makes data published in this submission inaccessible to anyone
```

Comment:
Proponuję: "Revoking a publication will render all data comprising this submission invisible to anyone except yourself. A revoked publication can be republished at a later date, if you choose to do so."